### DISCUSSION TITLE
ULI Pain Points

### DISCUSSION THREAD

**Discussion by jfbreault2022** [RESOStandards/uli-service#8] [Ideas]
Insert your ULI pain points here

**Comment by darnjo** [comment]
From the March 15, 2022 Meeting:
* Data shares - need to de-duplicate identifiers to know who is who between markets. 
* Statistical Reports - sales and broker stats need unique ids in order to produce accurate reports.
* Transactional History -  want to ensure that a stable identifier is used to track transactions. 
* Display - want to make sure that listings show the unique licensee who's listing them.
* Unified Educational Profile. Designations. Certification. There are cases where people have more than one national identifier and the goal is link them together.
* Licensee Interoperability Among Products. Single access to MLS and Association forms. Interoperability between showing and tour products and the MLS. 
* Is NRDS PII? CREA? What are the usage and display rules of these identifiers?
* Updating and notifying participants of changes in a timely and convenient manner.
* Human error.

**Reply by RoyaChaudhry** [reply]
Should bullet point 4 - Display include "selling"  Display - want to make sure that listings show the unique licensee who's listing or selling them.

**Reply by darnjo** [reply]
Good question, Roya. We had discussed the ULI being more of a background data element rather than something for display, but it definitely could be.

**Comment by etieseler** [comment]
A couple more pain points/use cases:

- MetroList vends MLS services for various Associations who maintain their own AMS systems, with ULI MetroList will be able to match the members our AMS systems since we do not track and maintain NRDS/M1.

- Merging with or acquiring neighboring MLS systems where a significant portion of membership overlaps.
